### DISCUSSION TITLE
Nuova Categoria - Parlano di Noi

### DISCUSSION THREAD

**Discussion by gbvitrano** [opendatasicilia/opendatasicilia.it#38] [Domande]
Per favore mi date i link ad articoli e post che citano Open Data Sicilia, cosi rinnoviamo la sezione

**Comment by gbvitrano** [comment]
Ho trovato questi link dove si parla di ODS, c'è ne sono altri?


<p><strong>2021</strong> | <a href="https://www.economysicilia.it/open-data-sicilia-open-data-garantiscono-trasparenza-e-rispetto-per-i-cittadini/">Open Data Sicilia: "Open Data garantiscono trasparenza e rispetto per i cittadini" (economysicilia.it)</a></p>



<p><strong>2021</strong> | <a href="https://www.ilfattonisseno.it/2021/10/caltanissetta-dati-covid-in-tempo-reale-con-la-web-app-di-open-data-sicilia/">Caltanissetta, dati Covid in tempo reale con la "web app" di Open Data Sicilia - il Fatto Nisseno - Caltanissetta notizie, cronaca, attualità</a></p>



<p><strong>2021</strong> | <a href="http://www.telepatti.it/News.asp?Azione=Dettaglio&amp;Id=6644#">Open Data Sicilia presenta on-line il progetto 'Informa Covid' (telepatti.it)</a></p>



<p><strong>2021</strong> | <a href="https://qds.it/open-data-sicilia-in-passato-gia-segnalati-errori-nei-dati-covid/">Open Data Sicilia, in passato già segnalati errori nei dati Covid - QdS</a></p>



<p><strong>2021</strong> | <a href="https://www.youtube.com/watch?v=8Tp3C0in_j4" data-type="URL" data-id="https://www.youtube.com/watch?v=8Tp3C0in_j4" target="_blank" rel="noreferrer noopener">TgMed del 01/04/2021 ore 13.00 - TRM Web - Palermo</a></p>



<p><strong>2021</strong> | <a href="https://24live.it/2021/04/15/la-mappa-dei-comuni-siciliani-in-zona-rossa-aggiornata-con-opendatasicilia/">La mappa dei comuni siciliani in zona rossa aggiornata con OpendataSicilia - 24live.it</a></p>



<p><strong>2021</strong> | <a href="https://www.sardegnareporter.it/2021/03/dati-covid-alterati-open-data-sicilia-nostre-segnalazioni-nel-vuoto/383113/">Dati Covid alterati, Open Data Sicilia: “nostre segnalazioni nel vuoto” - Sardegna Reporter</a></p>



<p><strong>2021</strong> | <a href="https://www.datibenecomune.it/2021/04/01/lappello-di-open-data-sicilia-per-dati-pubblici-accessibili-a-tutti/">L'appello di Open Data Sicilia per dati pubblici accessibili a tutti - #datiBeneComune</a></p>



<p><strong>2019</strong> | <a href="https://normanno.com/attualita/foiapop-il-portale-per-controllare-le-istituzioni-sicilia/">FoiaPOP, il portale per "controllare" le istituzioni made in Sicilia - Normanno.com</a></p>



<p><strong>2019</strong> | <a href="https://palermo.mobilita.org/2019/03/19/la-mappa-navigabile-degli-incidenti-a-palermo-nel-2018-a-cura-di-opendatasicilia/">La mappa navigabile degli incidenti a Palermo nel 2018 (a cura di OpenDataSicilia) - Mobilita Palermo</a></p>



<p><strong>2019 </strong>| <a href="https://www.slideshare.net/libreitalia/opendata-un-percorso-tra-giurisprudenza-formati-e-strumenti-e-modalit-di-pubblicazione-andrea-borruso-mario-grimaldi-e-ciro-spataro-opendata-sicilia">OpenData, un percorso tra giurisprudenza, formati e strumenti e modal… (slideshare.net)</a></p>



<p><strong>2018</strong> | <a href="https://www.ingenium-magazine.it/open-data-e-linked-data-risorsa-per-collettivita/">Open Data e Linked Data risorsa per la collettività? – ingenium (ingenium-magazine.it)</a></p>



<p><strong>2018</strong> | <a href="https://www.forumpa.it/citta-territori/open-data-sicilia-presenta-il-progetto-openars/">"Open Data Sicilia" presenta il progetto OpenARS - FPA (forumpa.it)</a></p>



<p><strong>2018</strong> |  <a href="https://www.forumpa.it/open-government/open-data/imparare-dalle-comunita-opendatasicilia-e-la-cultura-del-dato-sul-territorio/">Imparare dalle comunità. OpenDataSicilia e la cultura del dato sul territorio - FPA (forumpa.it)</a></p>



<p><strong>2018</strong> |  <a href="https://open.gov.it/premio-2018/premiazione-2018/">Premiazione OpenGov Champion 2018 - Open Government Partnership Italia</a></p>



<p><strong>2018</strong> |  <a rel="noreferrer noopener" href="http://copernicus.eu/news/copernicus-free-open-source-community" target="_blank">Copernicus and the free &amp; open source software community</a></p>



<p><strong>2017</strong> |  <a href="https://www.balarm.it/news/magazine/palermo-viaggia-nel-tempo-fino-al-1935-grazie-a-una-mappa-digitale-georeferenziata-18336">Palermo viaggia nel tempo fino al 1935, grazie a una mappa digitale georeferenziata (balarm.it)</a></p>



<p><strong>2017</strong> | <a href="https://www.ilsicilia.it/online-la-prima-mappa-delle-notizie-sugli-incendi/">Italiaafuoco.info, ecco la prima mappa delle notizie sugli incendi :ilSicilia.it</a></p>



<p><strong>2017</strong> | <a href="https://www.ingenium-magazine.it/open-data-e-incendi-il-progetto-italia-a-fuoco/">Open Data e incendi: il progetto “Italia a fuoco” – ingenium (ingenium-magazine.it)</a></p>



<p><strong>2017</strong> | <a href="https://www.agi.it/data-journalism/sicilia_banda_larga-2122269/news/2017-09-06/">La Sicilia: recuperare il divario digitale ed economico con la banda ultralarga (agi.it)</a></p>



<p><strong>2016</strong> | <a href="https://www.forumpa.it/open-government/open-data/open-data-sicilia-esperienze-di-contaminazioni-culturali/">Open Data Sicilia: esperienze di contaminazioni culturali - FPA (forumpa.it)</a></p>



<p><strong>2015</strong> | <a href="https://www.restoalsud.it/talenti/ecco-chi-sono-in-sicilia-e-in-sardegna-gli-innovatori-dellopen-data/">Ecco chi sono in Sicilia e in Sardegna gli innovatori dell'open data (restoalsud.it)</a></p>



<p><strong>2015</strong> | <a href="https://meridionews.it/articolo/37292/open-data-i-cittadini-sostituiscono-la-regione-nasce-il-portale-non-ufficiale-per-la-trasparenza/">Open data, i cittadini sostituiscono la Regione. Nasce il portale non ufficiale per la trasparenza - MeridioNews</a></p>



<p><strong>2015</strong> | <a href="http://www.consorzioarca.it/index.php/it/notizie-news/item/580-ad-arca-il-primo-raduno-di-open-data-sicilia/580-ad-arca-il-primo-raduno-di-open-data-sicilia">Ad ARCA il primo raduno di Open Data Sicilia - Consorzio ARCA</a></p>

**Comment by dennisangemi** [comment]
Ciao @gbvitrano , grazie mille. Anche io stavo pensando di aprire una discussion su questo tema. Hai a portata di mano tutti gli articoli che ci hanno citato in occasione del comunicato stampa https://adesso.opendatasicilia.it ? Sono tutti raccolti [qui](https://docs.google.com/spreadsheets/d/1O32BT_zDCLYAhADsT8lfBseKvEPleLgRRcH-G5VPguI/edit#gid=0)

**Comment by dennisangemi** [comment]
**2021** |[ Il programmatore siciliano che sta liberando i dati comunali del Covid-19 - VICE](https://www.vice.com/it/article/wx5ekx/sicilia-dati-comuni-covid)

**Reply by dennisangemi** [reply]
**2021** | [Coronavirus Serradifalco, striscia quotidiana sui dati del contagio in collaborazione con il comune - La Voce del Nisseno](https://www.lavocedelnisseno.it/coronavirus-serradifalco-striscia-quotidiana-sui-dati-del-contagio-in-collaborazione-con-il-comune/)

**Comment by gbvitrano** [comment]
@dennisangemi tutti i link li ho aggiunti, grazie grazie :-)
